packages/binutils/build.sh

```diff
@@ -41,8 +41,30 @@ termux_step_post_make_install() {
 	cd $TERMUX_PREFIX/bin
 	# Setup symlinks as these are used when building, so used by
 	# system setup in e.g. python, perl and libtool:
-	for b in ar ld nm objdump ranlib readelf strip; do
+	for b in ar nm objdump ranlib readelf strip; do
 		ln -s -f $b $TERMUX_HOST_PLATFORM-$b
 	done
+	ln -sf ld.bfd $TERMUX_HOST_PLATFORM-ld
```

Comment:
Can you move this to the lld package and have this `$TERMUX_HOST_PLATFORM-ld` symbolic link redirect to lld instead? The NDK only uses lld now, so we should do the same.

Reply:
Ping, if you make this change, we can get this in.

Reply:
Ping again, if you update this, we can get it in, @xtkoba.